src/Ui/Table/Command/GetTableEntries.php

```diff
@@ -56,7 +56,7 @@ public function handle()
         /*
          * Resolve the model out of the container.
          */
-        $repository = $this->builder->getRepository();
+        $repository = app($this->builder->getRepository());
```

Comment:
```php
if (is_string(($repository = $this->builder->getRepository()))) {
    $repository = app($repository);
}
```

Reply:
Yes, I know. I was too fast and didn't check.